integration-tests/pipelines/konflux-e2e-tests.yaml

```diff
@@ -0,0 +1,226 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: konflux-e2e-tests
+spec:
+  description: |-
+    This pipeline automates the process of running end-to-end tests for Konflux
+    using a ROSA (Red Hat OpenShift Service on AWS) cluster. The pipeline provisions
+    the ROSA cluster, installs Konflux using the infra-deployments, runs the image-controller tests, collects artifacts,
+    and finally deprovisions the ROSA cluster.
+  params:
+    - name: SNAPSHOT
+      description: 'The JSON string representing the snapshot of the application under test.'
+      default: '{"components": [{"name":"test-app", "containerImage": "quay.io/example/repo:latest"}]}'
+      type: string
+    - name: test-name
+      description: 'The name of the test corresponding to a defined Konflux integration test.'
+      default: ''
+    - name: ocp-version
```

Comment:
Is it bumped automatically?

Reply:
unfortunetly no. We need to stick to konflux version